src/main/java/net/snowflake/client/jdbc/SnowflakePreparedStatementV1.java

```diff
@@ -955,4 +950,20 @@ public boolean isAlreadyDescribed() {
   public boolean isArrayBindSupported() {
     return this.preparedStatementMetaData.isArrayBindSupported();
   }
+
+  @Override
+  public void resultSetHandler(SFBaseResultSet resultSet) throws SQLException {
```

Comment:
Can we rename as resultSetMetadataHandler? As such this routine is not doing anything else.

Reply:
done!